community/server/src/docs/ops/security.asciidoc

```diff
@@ -52,15 +52,13 @@ The Neo4j server includes built in support for SSL encrypted communication over
 The first time the server starts, it automatically generates a self-signed SSL certificate and a private key.
 Because the certificate is self signed, it is not safe to rely on for production use, instead, you should provide your own key and certificate for the server to use.
 
-To provide your own key and certificate, replace the generated key and certificate, or change _neo4j.conf_ to set the location of your certificate and key:
+To provide your own key and certificate, replace the generated key and certificate, in the directory where certificates are stored.  You can change _neo4j.conf_ to set the directory containing your certificate and key:
 
 [source,properties]
 ----
-# Certificate location (auto generated if the file does not exist)
-dbms.security.tls_certificate_file=certificates/snakeoil.cert
+# Certificates location (auto generated if the file does not exist)
+dbms.directories.certificates=certificates
 
-# Private key location (auto generated if the file does not exist)
-dbms.security.tls_key_file=certificates/snakeoil.key
```

Comment:
General documentation feedback (agreed with Docs team):
* Refer to files and directories by their default name (like "`_certificates_`" or "`_neo4j.conf_`") rather than using phrases like "in the directory where certificates are stored" or "the Neo4j configuration file".
* On pages that refer to `neo4j.conf`, put in one link to `[[file-locations]]` for the first mention on the page like this: `<<file-locations, _neo4j.conf_>>`. I suggest that we do the same thing for directories like `certificates`.
* Don't use "you".

More specifically:
* You need to explicitly mention the names of the files that must be provided.
* "Auto generated if the file does not exist" doesn't make sense as a comment on the directory setting.
* I'm not sure that the configuration fragment really deserves a spot here any more, since it's now unlikely that it will be modified just in order to provide a certificate.

I'd replace everything you've changed above with something like:

```
To override the send-signed key and certificate, replace _neo4j.key_ and _neo4j.cert_ in the <<file-locations, _certificates_>> directory. The location of this directory can be configured by setting _dbms.directories.certificates_ in <<file-locations, _neo4j.conf_>>.
```